README.md

```diff
@@ -152,14 +163,14 @@ Use the following command to grant permissions to each developer that will need
 
 ```bash
 
-# get the object id for each developer (optional)
-export dev_Object_Id=$(az ad user show --id {developer email address} --query objectId -o tsv)
-
-# grant Key Vault access to each developer (optional)
-az keyvault set-policy -n $He_Name --secret-permissions get list --key-permissions get list --object-id $dev_Object_Id
+# get the object ID by email address for each developer
+az keyvault set-policy -n $He_Name --secret-permissions get list --key-permissions get list --object-id \
+$(az ad user show --id {developer email address} --query objectId -o tsv)
 
 ```
 
+## TODO - per earlier suggestion, we could use the language repo instead of helium and skip this step
```

Comment:
Should this be a comment so that it doesn't get rendered to the reader?

Reply:
it's for our design review tomorrow